src/Endpoints/Suppression.php

```diff
@@ -67,8 +67,12 @@ public function create(SuppressionParams $params): array
      * @throws \MailerSend\Exceptions\MailerSendAssertException
      * @throws \Psr\Http\Client\ClientExceptionInterface
      */
-    public function delete(?array $ids = null, bool $all = false): array
+    public function delete(string $domainId, ?array $ids = null, bool $all = false): array
```

Comment:
Domain ID is not mandatory, you can delete all account suppression.
Also, we should move domain ID to the end of method parameters because it may break integrations.